src/proxyWorker.js

```diff
@@ -460,6 +466,9 @@ onmessage = function onmessage(message) {
       screen.height = Module.canvas.height_ = message.data.height;
       Module.canvas.boundingClientRect = message.data.boundingClientRect;
       document.URL = message.data.URL;
+#if USE_PTHREADS
+      currentScriptUrl = message.data.currentScriptUrl;
```

Comment:
It intends to replace this https://github.com/kripken/emscripten/blob/master/src/shell.js#L71.